docs/en/user-guide.md

```diff
@@ -0,0 +1,42 @@
+# Dumpling manual
+
+[Dumpling](https://github.com/pingcap/dumpling) is a tool that supports export MySQL/TiDB data as SQL text or CSV file.
+
+It's designed to replace [Mydumper](https://github.com/pingcap/mydumper), so you can learn its basic usage from Mydumper. Of course, we didn't make a replica of Mydumper, so there are difference between them.
```

Comment:
This paragraph sounds not so professional. Perhaps just copy the statement from the README.